### ISSUE TITLE
please post inference time for your examples

### ISSUE THREAD

**Issue by yosun** [Quantuman134/TexPainter#2]
please post inference time for your examples

**Comment by Quantuman134** [comment]
Each example takes around 25 minutes

**Comment by yosun** [comment]
wow, and i thought TEXTure took too long at 2 minutes... 

is there a way to speed things up

**Comment by Quantuman134** [comment]
Yes, more than 80% time is spent on an optimization process. This optimization can be replaced by a VAE encoding. The cost is some artifacts will appear in the generated result. You can find the detailed illustration in the Ablantion study and conclusion from the paper.
